### PR TITLE
new(iam): reproduce system before iam

### PR DESCRIPTION
**Pull request by crlptl**
Doc review internally, about how to reproduce roles and project-scoped API keys with IAM
I am not sure I managed navigation correctly, don't hesitate to give feedback

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+<Lightbox src="scaleway-iam-roles-change.webp" alt="" />
+
+You can keep adding users to these groups if you want to preserve the “old” system of roles and their corresponding rights. Make sure that you therefore do not delete the groups that were created during the migration process.
+
+<Lightbox src="scaleway-iam-roles-change.webp" alt="" />
+
+If you previously deleted these groups from your Organization, you can recreate them by following these steps:
+
+1. [Create a group](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-group/) from the Groups tab, preferably with a name related to the corresponding role (e.g. “Administrators”, “Editors” or “Billing Administrators”)
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx:35]:
```suggestion
1. [Create a group](/identity-and-access-management/iam/how-to/create-group/) from the Groups tab, preferably with a name related to the corresponding role (e.g. “Administrators”, “Editors” or “Billing Administrators”)
```

File: identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+2. [Create a policy](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-policy/) from the Policies tab, attached to the group you have just created and with the appropriate permission sets:
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx:37]:
```suggestion
2. [Create a policy](/identity-and-access-management/iam/how-to/create-policy/) from the Policies tab, attached to the group you have just created and with the appropriate permission sets:
```

File: identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+| Former role | Permission to add and Scope |
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx:39]: This table needs to be fixed, not displaying correctly. Replace with the following code.

| Former role           	| Necessary Permission Set                              	| Scope for the Permission Set                         	|
|-----------------------	|-------------------------------------------------------	|------------------------------------------------------	|
| Administrator         	| `OrganizationManager` <br><br>`AllProductsFullAccess` 	| Organization <br><br>All and current future Projects 	|
| Editor                	| `AllProductsFullAccess`                               	| All current and future Projects                      	|
| Billing Administrator 	| `BillingManager` <br><br>`OrganizationReadOnly`       	| Organization <br><br>Organization                    	|

File: identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+| Administrator | OrganizationManager on the Organization
+                  AllProductsFullAccess on All current and future projects |
+| Editor | AllProductsFullAccess on All current and future projects |
+| Billing Administrator | BillingManager on the Organization
+                          OrganizationReadOnly on the Organization |
+
+
+## How can I reproduce Project-scoped API keys?
+
+When your Organization was migrated to IAM, for each Project existing at the time of the migration, an [application](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#application) and a [policy](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#policy) were automatically created. These were both given the same name: `Project - [name of the project] (id of the project)`
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx:49]:
```suggestion
When your Organization was migrated to IAM, for each Project existing at the time of the migration, an [application](/identity-and-access-management/iam/concepts/#application) and a [policy](/identity-and-access-management/iam/concepts/#policy) were automatically created. These were both given the same name: `Project - [name of the project] (id of the project)`
```

File: identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+If you want to create Project-scoped API keys for Projects created after your Organization migrated to IAM, follow these steps:
+1. [Create an application](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-application/), preferably with a name related to the Project.
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx:52]:
```suggestion
1. [Create an application](/identity-and-access-management/iam/how-to/create-application/), preferably with a name related to the Project.
```

File: identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+2. [Create a policy](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-policy/) from the policy tab, attached to the application you just created and with the permission set AllProductsFullAccess for the Project in question.
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx:53]:
```suggestion
2. [Create a policy](/identity-and-access-management/iam/how-to/create-policy/) from the policy tab, attached to the application you just created and with the permission set `AllProductsFullAccess` for the Project in question.
```

File: identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+Going forward, whenever you wish to create a new API key with full access to this Project, you can simply go to the application’s overview page, click the API keys tab, and create a new key. 
+
+<Message type="important">
+If this API key will be used for Object Storage, make sure to select the Project in question as its [preferred Project for Object Storage](https://www.scaleway.com/en/docs/identity-and-access-management/iam/api-cli/using-api-key-object-storage/) when creating the key.
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx:58]:
```suggestion
If this API key will be used for Object Storage, make sure to select the Project in question as its [preferred Project for Object Storage](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/) when creating the key.
```

**Pull request merged**